pkg/controllers/manifestwork.go

```diff
@@ -309,7 +314,7 @@ func (r *HypershiftDeploymentReconciler) appendHostedClusterReferenceSecrets(ctx
 					Namespace: hyd.GetNamespace()})
 
 			if err != nil {
-				return fmt.Errorf("failed to duplicateSecret, err %w", err)
+				return fmt.Errorf("failed to duplicate ssh secret, err %w", err)
```

Comment:
I think the original message was wrong, the user won't understand what duplicating the secret means since we are actually just getting the secret resource to put in manifest work. The error should be returned directly, or at least logged, so the user can tell if the Client.Get failed or the duplication func failed.